README.md

```diff
@@ -35,6 +43,11 @@ Example Playbook
 - hosts: localectl-servers
   roles:
     - role: oasis-roles.localectl
+      localectl_system_locale: LANG=en_US.UTF-8
```

Comment:
At least on my system, this is not a valid locale name. Should be `en_US.utf8`

Reply:
I think this is OK.  If you do a
#localectl list-locales
It will list the locales without LANG=
Setting the locale requires the LANG= as per man pages
set-locale LOCALE...
           Set the system locale. This takes one or more assignments such as
           "LANG=de_DE.utf8", "LC_MESSAGES=en_GB.utf8", and so on. See
           locale(7) for details on the available settings and their meanings.
           Use list-locales for a list of available locales (see below).

 
Manual testing proves this.   Automation fails also when not given the LANG= prefix.

[root@test-localectl ~]# localectl set-locale en_US.UTF-8
Failed to issue method call: Invalid Locale data.
[root@test-localectl ~]# localectl set-locale LANG=en_US.UTF-8
[root@test-localectl ~]#

Reply:
This is about consistency with localectl, though, not what localectl will or will not accept. I don't think it matters what the value actually is, as long as whatever we put in the example for `set-locale` is consistent with the output of `list-locales`.